devicetypes/smartthings/zigbee-smoke-sensor.src/zigbee-smoke-sensor.groovy

```diff
@@ -129,7 +154,8 @@ def getDetectedResult(value) {
 def refresh() {
 	log.debug "Refreshing Values"
 	def refreshCmds = []
-	refreshCmds += zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021) +
+	def batteryAttr = isFrientSensor() ? 0x0020 : 0x0021
```

Comment:
We should try to reference the clusters, commands, and attributes using a constant name (like https://github.com/SmartThingsCommunity/SmartThingsPublic/blob/master/devicetypes/smartthings/zigbee-battery-accessory-dimmer.src/zigbee-battery-accessory-dimmer.groovy#L67) when possible. I know the DTH referenced `0x0021`, but I think we should use a named constant to help avoid confusion.